firmware/user/display/oled.c

```diff
@@ -451,61 +695,34 @@ bool ICACHE_FLASH_ATTR initOLED(bool reset)
 
 /**
  * Set all the parameters on the OLED for normal operation
- * This takes ~0.9ms to execute
+ * This takes ~1.3ms to execute on boot, and slightly less if restarting.
```

Comment:
0.4ms noooo!!!! (kidding)

Reply:
I'm not sure when you got it in 0.9ms.  It's not my code.